en/tutorials-and-examples/cms/authentication.rst

```diff
@@ -92,6 +92,8 @@ AuthComponent in our AppController::
                     'controller' => 'Users',
                     'action' => 'login'
                 ],
+                 //use isAuthorized in Controllers
+                'authorize' => array('Controller'),
```

Comment:
Do you know if the `isAuthorized` method is implemented as part of the tutorial?

Reply:
Yes it is on line 241 to 263